### ISSUE TITLE
Undocumented flags for convert4qmc

### ISSUE THREAD

**Issue by jtkrogel** [QMCPACK/qmcpack#672]
Quick question/discussion about convert4qmc.  Are the following correct?

1. -add3BodyJ is deprecated.
2. -zeroCi and -readInitialGuess are functional but not documented in the main converter section of the manual (but they are in section 18.7).
3. -orderByExcitation, -cutoff, -debug are functional but not documented anywhere.

**Comment by anbenali** [comment]
>> -add3BodyJ is deprecated.
True!! It is now by default. I will push it with the next PR for PBC gaussian. 

>>-zeroCi and -readInitialGuess, -orderByExcitation
Yes, they work and are part of the MCSCF Gamess part. Since I did not implement them and do not use them it would have been longer for me to write somethong about them so I left it for a generous soul who has 5 minutes and know them.

>> cutoff, debug.
My bad. I will update with the next PR.

**Comment by prckent** [comment]
Good questions.

@anbenali  Please keep PRs separate.

The problem with add3bodyJ is that we need to comment it out somehow otherwise the simple input is very unlikely to be optimized. This is not a friendly default and needs to be solved. The 3body Jastrow issue is noted in the manual, but for ease of use it will have to be commented out somehow in the example input. I think that to do so would require writing the XML files "by hand" instead of using libxml2, since the latter does not support the level of commenting we need to create friendly inputs.

**Comment by jtkrogel** [comment]
I have no plans to use add3BodyJ.  Right now I am updating Nexus to be in sync with the current converter (supporting all user input) and to open a pathway for automated -hdf5 style runs.
